tests/blockchain/eth/contracts/integration/test_intercontract_integration.py

```diff
@@ -104,6 +106,30 @@ def policy_manager(testerchain, escrow):
     return contract, dispatcher
 
 
+@pytest.fixture()
+def overseer(testerchain, escrow):
+    escrow, _ = escrow
+    creator = testerchain.interface.w3.eth.accounts[0]
+
+    # secret_hash = testerchain.interface.w3.sha3(policy_manager_secret)
+
+    # Creator deploys the contract
+    #  TODO change to the normal contract
+    contract, _ = testerchain.interface.deploy_contract('ChallengeOverseerForMinersEscrowMock', escrow.address)
+    # dispatcher, _ = testerchain.interface.deploy_contract('Dispatcher', contract.address, secret_hash)
+
+    # # Wrap dispatcher contract
```

Comment:
Looks like you are avoiding wrapping here, and using the mock instead?

Reply:
Fixed in #606